kube-client/src/client/mod.rs

```diff
@@ -35,6 +35,8 @@ mod config_ext;
 pub use auth::Error as AuthError;
 pub use config_ext::ConfigExt;
 pub mod middleware;
+
+#[cfg(feature = "ws")] mod node_proxy;
```

Comment:
should we set a new feature flag ?

Reply:
the only reason for this would be for potential instability. looking at [configuration docs upstream](https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/#kubelet-config-k8s-io-v1beta1-KubeletConfiguration) it does appear that the flag for these "debug handlers" is default enabled so i expect them to treat it with some respect (despite the name)

so given it doesn't introduce any new deps (and no significant compilation weight), it feels ok to me to lump it in with `ws`